File: SEP3_Client/Model/Unit/User/Birthday.cs
namespace SEP3_Client.Model.Unit.User
{
    public class Birthday
    {
        public int Month { get; set; }
        public int Day { get; set; }

        public Birthday(int day,int month)
        {
            Month = month;
            Day = day;
        }

        public int GetDay() {
            return Day;
        }

        public int GetMonth() {
            return Month;
        }

        public void Set(int day,int month)
        {
            Month = month;
            Day = day;
        }

        public Birthday Copy()
        {
            return new Birthday(Day,Month);
        }
    }
}

File: SEP3_Database/src/main/java/com/example/SEP3_Database/model/domain/unit/user/Birthday.java
package com.example.SEP3_Database.model.domain.unit.user;

public class Birthday {
    private int Month;
    private int Day;

    public Birthday(int day,int month)
    {
        Month = month;
        Day = day;
    }

    public Birthday()
    {
        Month = 0;
        Day = 0;
    }

    public boolean isSet()
    {
        return Month!=0&&Day!=0;
    }

    public int getDay() {
        return Day;
    }

    public int getMonth() {
        return Month;
    }

    public void set(int day,int month)
    {
        Month = month;
        Day = day;
    }

    public Birthday copy()
    {
        return new Birthday(Day,Month);
    }
}
